README.md

```diff
@@ -89,7 +89,27 @@ Using the bot to manage your rooms is great, however if you want to use your ban
 is needed. Primarily meant to block invites from undesired homeservers/users, Mjolnir's
 Synapse module is a way to interpret ban lists and apply them to your entire homeserver.
 
-First, install the module to your Synapse python environment:
+**Warning**: This module works by running generated Python code in your homeserver. The code
+is generated based off the rules provided in the ban lists, which may include crafted rules
+which may allow unrestricted access to your server. Only use lists you trust and do not
+use anyone else's infrastructure to get the ruleset from - only use infrastructure that
+you control.
+
+If this is acceptable, the following steps may be performed.
+
+First, run the Docker image for the rule server. This is what will be serving the generated
+Python for the Synapse antispam module to read from. This rule server will serve the Python
+off a webserver at `/api/v1/py_rules` which must be accessible by wherever Synapse is installed.
```

Comment:
From my testing, the rules server appears to serve the rules from any endpoint, not just `/api/v1/py_rules`.

Reply:
yea, that's technically a bug but the documentation should be what people expect to work.